addon/helpers/number.js

```diff
@@ -3,6 +3,10 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([num = 0]) {
+    return num;
```

Comment:
While we're in the business of coercion, let's go ahead an coerce to a number with `Number(num)`